sessionctx/variable/varsutil.go

```diff
@@ -137,6 +137,25 @@ func SetSessionSystemVar(vars *SessionVars, name string, value types.Datum) erro
 	return vars.SetSystemVar(name, sVal)
 }
 
+// ValidateGetSystemVar check if system variable exists and validate it's scope when get system variable.
+func ValidateGetSystemVar(name string, isGlobal bool) error {
+	sysVar, exists := SysVars[name]
+	if !exists {
+		return UnknownSystemVar.GenByArgs(name)
+	}
+	switch sysVar.Scope {
+	case ScopeGlobal, ScopeNone:
```

Comment:
why handle `ScopeGlobal` and `ScopeNone` together?

Reply:
`ScopeNone` variables are global variables that can't be modified dynamically